sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/models/data-inputs/data-inputs.models.ts

```diff
@@ -81,6 +81,9 @@ export interface IER12WorkflowData {
     livingWithParents: FormYesNoOptions;
     numberOfParents?: number;
   };
+  dmnValues: {
```

Comment:
IE12 is intended to be full-time only from a business perspective. Does it need to be required here?